packages/web/src/helpers/_links.scss

```diff
@@ -0,0 +1,33 @@
+@use 'sass:map';
+@use '../settings/cursors';
+@use '../settings/links' as links-settings;
+@use '../tools/links';
+
+@include links.variants('.link', links-settings.$variants, (default, hover, active, disabled));
+
+// All disabled links have default cursor
```

Comment:
That's quite a brave statement, from here you don't know what's the value of `cursor.$disabled` 🙂.

Reply:
🤦‍♂️